components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { Button } from '../Button/Button';
+import { cn } from '@/utils/utils';
+import { EntryStatus } from '../EntryStatus/EntryStatus';
+import { ILeagueEntriesProps } from './LeagueEntries.interface';
+import * as React from 'react';
+
+const LeagueEntries = React.forwardRef<HTMLDivElement, ILeagueEntriesProps>(
+  ({ entryNumber, isEliminated = false, isPickSet = false }, ref) => (
+    <div
+      data-testid="LeagueEntryContainerCard"
```

Comment:
Same with testid's they should be lowercase and hyphen separated.

Reply:
Updated in latest commit.